crates/egui-winit/src/lib.rs

```diff
@@ -1310,11 +1344,14 @@ fn process_viewport_command(
         ViewportCommand::InnerSize(size) => {
             let width_px = pixels_per_point * size.x.max(1.0);
             let height_px = pixels_per_point * size.y.max(1.0);
-            if window
-                .request_inner_size(PhysicalSize::new(width_px, height_px))
-                .is_some()
+            if let Some(_inner_size) =
+                window.request_inner_size(PhysicalSize::new(width_px, height_px))
             {
-                log::debug!("ViewportCommand::InnerSize ignored by winit");
+                log::info!("ViewportCommand::InnerSize ignored by winit");
```

Comment:
I don't like this log message.

Would it make sense to pass the current viewport size into `process_viewport_command`?

It would allow for a more accurate message.
Also adding `resize_requested: &mut bool` and comparing sizes on `InnerSize` might preferable to comparing the before and after viewport size for every group of commands.

Reply:
This is to support both `glow` and `wgpu`.